controllers/storageconsumer/consumer_test.go

```diff
@@ -179,18 +147,13 @@ func TestCephName(t *testing.T) {
 				{
 					Kind:  "CephClient",
 					Name:  "rbd",
-					Phase: "Error",
+					Phase: "Ready",
```

Comment:
we have only Noobaa in the status nothing else, right? why do we have ceph clients in the status field?

Reply:
Removed all cephclients